config/config/wc-config.yaml

```diff
@@ -4,7 +4,12 @@ global:
   clusterName: ${CK8S_ENVIRONMENT_NAME}-wc
   baseDomain: "set-me"
   opsDomain: "set-me"
+
+  ## Default cert-manager issuer to use in ingresses
   issuer: letsencrypt-staging
+
+  ## Verify ingress certificates
+  ##
   verifyTls: true
   clusterDns: "10.43.0.10"
```

Comment:
How do you decide on what variables to comment? I'm guessing you only comments the lines you feel necessary, but I'm curios if you have any process for deciding.